src/_data/side-nav.yml

```diff
@@ -1,8 +1,10 @@
-- title: Samples & tutorials
+- title: Tutorials & codelabs
```

Comment:
**issue:** Codelabs are tutorials. Calling this "Tutorials" would be recommended.
```suggestion
- title: Tutorials
```

I understand that this would result in some refactoring after this heading. I would encourage that, too.

Reply:
What do you have in mind? Now with this change there is **Tutorials** under **Tutorials**

Reply:
I think that change needs to be it's own PR, because the necessary reorg under the heading (that @parlough is referring to) needs it's own work and discussion that's blocking this PR.